src/tools/cephfs/cephfs-shell

```diff
@@ -876,16 +876,21 @@ class CephFSShell(Cmd):
                 continue
             else:
                 is_pattern = False
-            path = os.path.normpath(os.path.join(cephfs.getcwd(), path))
+
             if args.parent:
-                files = reversed(sorted(set(dirwalk(path))))
+                path = os.path.join(cephfs.getcwd(), path.rsplit(b'/')[0])
+                files = list(reversed(sorted(set(dirwalk(path)))))
```

Comment:
calling`reversed` and `sorted` isn't much useful, AFAIS.

Reply:
True, I will update it.